doc/procedures/pre_commit.rst

```diff
@@ -47,6 +47,32 @@ An example where all ``pre-commit`` hooks are enabled is:
    isort: true
    mypy: true
 
+
+The ``generate_pre_commit_conf`` command fails with a comprehensive error message if a mandatory or optional ``pre-commit`` hook is missing.
+All ``pre-commit`` hooks except for ``mypy`` are mandatory for TSSW projects, so those hooks need to be set to ``true``.
+Setting one or more of the mandatory hooks to ``false`` will make the ``generate_pre_commit_conf`` command fail with a comprehensive error message.
+
+The configuration files will be updated whenever the pre-commit hooks get updated.
+Apart from that, all generated configuration file names get added to ``.gitignore`` as well.
+The TSSW Jenkins CI jobs update these configuration files and execute pre-commit every time the jobs run, so you know that you need to update them locally and fix your code if a job fails.
+You can update the config file locally by running the ``generate_pre_commit_conf`` command again.
+
+Projects with a ``.ts_pre_commit_config.yaml`` file
+---------------------------------------------------
+
+There are several situations in which a project already may have a ``.ts_pre_commit_config.yaml`` file.
+One is when another developer has configured the project and you have cloned the repo.
+Another is when a project was configured by you and one or more ``pre-commit`` hooks were updated.
+A third is when the project previously didn't use ``mypy`` and now it does.
+
+In both cases the config files for ``pre-commit`` and the hooks can easily be created or updated.
```

Comment:
`both` implies two, but you list three cases. Maybe should be `In all cases`.

Reply:
Thanks for catching that. I had two cases at first and then thought of a third. I'll change it to `In all cases`.